src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, value) {
+      operation(value);
+
+      return this;
+    },
+
+    add(value) {
+      calculator.result += value;
```

Comment:
you can use this to access object properties
```suggestion
      this.result += value;
```